Doc/library/codecs.rst

```diff
@@ -1476,6 +1476,65 @@ mapping. It is not supported by :meth:`str.encode` (which only produces
    Restoration of the ``rot13`` alias.
 
 
+:mod:`encodings` --- Encodings package
+--------------------------------------
+
+.. module:: encodings
+   :synopsis: Encodings package
+
+This module implements the following functions:
+
+.. note::
+   These functions should not be used directly, except for testing purposes;
+   :func:`codecs.lookup` should be used instead.
```

Comment:
I don't think that it's the right place to mention codecs.lookup(). You cannot replace normalize_encoding() with codecs.lookup() for example.